src/core/components/fields/LabeledTextField.tsx

```diff
@@ -15,22 +15,20 @@ export interface LabeledTextFieldProps extends PropsWithoutRef<JSX.IntrinsicElem
 
 export const LabeledTextField = forwardRef<HTMLInputElement, LabeledTextFieldProps>(
   ({ name, label, outerProps, fieldProps, labelProps, ...props }, ref) => {
+    let validValue = (v) => (v === "" ? null : v)
+    let myType = props.type === "number" ? (Number as any) : validValue
```

Comment:
If you are changing core code, even for improvement, please do not do so without explaining why. I can understand the `data-testid`changes, but why this change is here less so. 

Can you expand?

Reply:
This “transform error” was commented on Slack and the suggestion was to do the fix on the test.
Basically, when I run the test, I get the following error:
src/core/components/fields/LabeledTextField.tsx:35:28: ERROR: Expected ":" but found ",".

So, I think it is due to an extra comma that is added by the vscode linter or formatter in line 26. The solution was to move the code to a variable to avoid the extra comma. The original expression to assign the value to the parse property is exactly the same, except that now it is assigned to a variable first. 
We probably need to create an issue to refactor those two new lines of code (18,19) into a function since it is also found in other components.